Failures on Social Login Tests

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

```
[ERROR] Failures: 
2024-06-26 10:35:22  [ERROR]   SocialLoginTest.bitbucketLogin:332 Expected AppPage but was Log in with Atlassian account (https://id.atlassian.com/login/security-screen?token=eyJraWQiOiJtaWNyb3MvaWQtYXV0aGVudGljYXRpb24vcTQ4ZzhmOWZhMnEwdjRzciIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJtaWNyb3MvaWQtYXV0aGVudGljYXRpb24iLCJjb250YWluZXJUeXBlIjoiZ2xvYmFsIiwiaXNzIjoibWljcm9zL2lkLWF1dGhlbnRpY2F0aW9uIiwicHJvdmlkZWRGYWN0b3JzIjpbIlBhc3N3b3JkIl0sInRyYW5zYWN0aW9uSWQiOiJ1cy1lYXN0LTF8NDNhNDlkYzEtZDJmYi00MmI3LWFmZmQtZGJjZjYzYTFiOTM5IiwiYWNjb3VudElkIjoiNzEyMDIwOmU5OTM4Y2ZkLTY3YjAtNGU1ZS05OTY1LWM1ZTliNDRmY2VkOSIsImF1ZCI6ImlkLWF1dGhlbnRpY2F0aW9uIiwibmJmIjoxNzE5MzkwNzE3LCJzY29wZSI6Ik1mYVByb21vdGUiLCJleHAiOjE3MTkzOTEzMTcsImlhdCI6MTcxOTM5MDcxNywianRpIjoidXMtZWFzdC0xfDBkNTQ0NWM5LWM2NWYtNGQ1YS04MDY4LTgyNDA0ZmYwMGYzYyIsImhhc2hlZENzcmZUb2tlbiI6IjZjNmI2ZWI5MmY5ZWQxOTVhZTlmYzM1M2VkMzljODg4OTM2YTVhMTYzZWZmYmJlNmZkY2U4OTFiMTE3OWRjZDQifQ.Nae4GqH_HYKl2rt4SR5Ciq6A7Ew8Fz5Wyw_vyBOp0DzafpWJDr0LAJPsE67wHxeB6anQKk14m0im1XyGnkxB5jt2zpD3jNrm2dzlMj7HoPj7Rg0vjQk8ttdKofcK72qlJsNvG1_aSyequ1Q94INVsG2ZTC-G7B6eMbubhZZ-UwSADWYb3wzdRfXwVCpGPu9nAMPccNgPLeHwwzGhV0MaLupFWVyYGqcbVVgff96cGuzRZMqJH3headRXNjsZV13RVdvZUjOnfpXHVN0nkKyANMp9zcjYs1czqofySlV_TyjGPJ8M_Mv2p-bQisxxyFKm0EqvSeNp9HawT1g8kRpy_A&continue=https%3A%2F%2Fbitbucket.org%2Faccount%2Fsignin%2F%3Fnext%3D%252Fsite%252Foauth2%252Fauthorize%253Fscope%253Daccount%2526state%253DpVjJwCLiv5MDJfwdVsqr-wG3J3J10MRtOgq2Dq9hU6s.2gc3nOr-kGw.vDT_lZIqR4-mwqVHvtnLeQ.eyJydSI6Imh0dHBzOi8vbG9jYWxob3N0Ojg1NDMvYXV0aC9yZWFsbXMvbWFzdGVyL2FwcC9hdXRoIiwicnQiOiJjb2RlIiwic3QiOiJlOTQwZTUxNi05NWFmLTRhNWUtOGViZS00NTA3ZWM3ZDc0MWIifQ%2526response_type%253Dcode%2526client_id%253DBWGdyFqB8n8hVDr9Rh%2526redirect_uri%253Dhttps%25253A%25252F%25252Flocalhost%25253A8543%25252Fauth%25252Frealms%25252Fsocial%25252Fbroker%25252Fbitbucket%25252Fendpoint%26redirectCount%3D1)
2024-06-26 10:35:22  [ERROR] Errors: 
2024-06-26 10:35:22  [ERROR]   SocialLoginTest.instagramLogin:369->assertUpdateProfile:583 Â» NoSuchElement no such element: Unable to locate element: {"method":"css selector","selector":"#firstName"}
2024-06-26 10:35:22    (Session info: chrome=126.0.6478.116)
2024-06-26 10:35:22  For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
2024-06-26 10:35:22  Build info: version: '4.21.0', revision: '79ed462ef4'
2024-06-26 10:35:22  System info: os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
2024-06-26 10:35:22  Driver info: org.openqa.selenium.chrome.ChromeDriver
2024-06-26 10:35:22  Command: [d5a93ad5429dc15b7e8fea256b2a7c03, findElement {using=id, value=firstName}]
2024-06-26 10:35:22  Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.116, chrome: {chromedriverVersion: 125.0.6422.141 (4b1e8393712..., userDataDir: C:\Users\hudson\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:53561}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:53561/devtoo..., se:cdpVersion: 126.0.6478.116, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
2024-06-26 10:35:22  Session ID: d5a93ad5429dc15b7e8fea256b2a7c03
2024-06-26 10:35:22  [ERROR]   SocialLoginTest.stackoverflowLogin:435->performLogin:514->doLogin:558 Â» ElementNotInteractable element not interactable
2024-06-26 10:35:22    (Session info: chrome=126.0.6478.116)
2024-06-26 10:35:22  Build info: version: '4.21.0', revision: '79ed462ef4'
2024-06-26 10:35:22  System info: os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
2024-06-26 10:35:22  Driver info: org.openqa.selenium.chrome.ChromeDriver
2024-06-26 10:35:22  Command: [d5a93ad5429dc15b7e8fea256b2a7c03, clickElement {id=f.678B4ECF59E92E0B727E7A1C2803C577.d.41167E10F0C9445A562D1D91A516951C.e.25}]
2024-06-26 10:35:22  Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.116, chrome: {chromedriverVersion: 125.0.6422.141 (4b1e8393712..., userDataDir: C:\Users\hudson\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:53561}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:53561/devtoo..., se:cdpVersion: 126.0.6478.116, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
2024-06-26 10:35:22  Element: [[[[ChromeDriver: chrome on windows (d5a93ad5429dc15b7e8fea256b2a7c03)] -> tag name: html]] -> xpath: //button[@name='submit-button']]
2024-06-26 10:35:22  Session ID: d5a93ad5429dc15b7e8fea256b2a7c03
2024-06-26 10:35:22  [ERROR]   SocialLoginTest.twitterLogin:395->performLogin:514->doLogin:558 Â» NoSuchElement no such element: Unable to locate element: {"method":"xpath","selector":"//input[@type='text' and @name='text']"}
2024-06-26 10:35:22    (Session info: chrome=126.0.6478.116)
2024-06-26 10:35:22  For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
2024-06-26 10:35:22  Build info: version: '4.21.0', revision: '79ed462ef4'
2024-06-26 10:35:22  System info: os.name: 'Windows Server 2016', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
2024-06-26 10:35:22  Driver info: org.openqa.selenium.chrome.ChromeDriver
2024-06-26 10:35:22  Command: [d5a93ad5429dc15b7e8fea256b2a7c03, findChildElement {id=f.678B4ECF59E92E0B727E7A1C2803C577.d.F6A39C3977033D6A445E622E731D155A.e.2, value=//input[@type='text' and @name='text'], using=xpath}]
2024-06-26 10:35:22  Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.116, chrome: {chromedriverVersion: 125.0.6422.141 (4b1e8393712..., userDataDir: C:\Users\hudson\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:53561}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:53561/devtoo..., se:cdpVersion: 126.0.6478.116, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
2024-06-26 10:35:22  Session ID: d5a93ad5429dc15b7e8fea256b2a7c03
2024-06-26 10:35:22  [INFO] 
2024-06-26 10:35:22  [ERROR] Tests run: 14, Failures: 1, Errors: 3, Skipped: 0
```

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/social/BitbucketLoginPage.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/social/BitbucketLoginPage.java
@@ -35,6 +35,9 @@ public class BitbucketLoginPage extends AbstractSocialLoginPage {
     @FindBy(id = "password")
     private WebElement passwordInput;
 
+    @FindBy(id = "mfa-promote-dismiss")
+    private WebElement dismissMFA;
+
     @FindBy(xpath = "//div[contains(@class,'additional-auths')]/p/a")
     private WebElement loginWithAtlassianButton;
 
@@ -54,5 +57,12 @@ public class BitbucketLoginPage extends AbstractSocialLoginPage {
 
         passwordInput.sendKeys(password);
         passwordInput.sendKeys(Keys.RETURN);
+
+        try {
+            clickLink(dismissMFA);
+        }
+        catch (NoSuchElementException e) {
+            log.info("MFA already dismissed");
+        }
     }
 }

